### ISSUE TITLE
has_axon in CheckRunner returns always False

### ISSUE THREAD

**Issue by eleftherioszisis** [BlueBrain/NeuroM#622]
```python
from neurom.check.runner import CheckRunner

config = {'checks': {'neuron_checks': ['has_axon']}}

check_runner = CheckRunner(config)

results = check_runner.run(directory_of_Neuron.swc)
```
returns False which is not true for the ```Neuron.swc``` in ```test_data```. Basically for every cell it returns False.

I tried the same with ```morph_check```, and that returned the expected correct values.

**Comment by eleftherioszisis** [comment]
Narrowing down a bit:
```
check_runner.run('Neuron.swc')
```
return the correct values while,
```
check_runner.run('directory_of_Neuron.swc')
```
returns the dict and ```has_axon``` is set to False this time.

**Comment by mgeplf** [comment]
Weird, I will look into that.

**Comment by eleftherioszisis** [comment]
Could it be related to the ```self.summary``` updating in ```CheckRunner```? Unfortunately, the juggling of this dict in the functions is hard to follow just by eye. :(

**Comment by mgeplf** [comment]
It's a lot easier when the bug report is as good as this one.  Thanks for that.
